streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -143,18 +135,16 @@ public KTableImpl(KStreamBuilder topology,
 
     @Override
     public KTable<K, V> through(String topic,
-                                Serializer<K> keySerializer,
-                                Serializer<V> valSerializer,
-                                Deserializer<K> keyDeserializer,
-                                Deserializer<V> valDeserializer) {
-        to(topic, keySerializer, valSerializer);
+                                SerDe<K> keySerDe,
+                                SerDe<V> valSerializationation) {
```

Comment:
What?  I think `valSerializationation` is a double-typo, and should be called `valueSerDe` (to be consistent with `keySerDe`.